README.md

```diff
@@ -68,11 +68,6 @@
 
 AutoGen is an open-source programming framework for building AI agents and facilitating cooperation among multiple agents to solve tasks. AutoGen aims to streamline the development and research of agentic AI, much like PyTorch does for Deep Learning. It offers features such as agents capable of interacting with each other, facilitates the use of various large language models (LLMs) and tool use support, autonomous and human-in-the-loop workflows, and multi-agent conversation patterns.
 
-**Open Source Statement**: The project welcomes contributions from developers and organizations worldwide. Our goal is to foster a collaborative and inclusive community where diverse perspectives and expertise can drive innovation and enhance the project's capabilities. Whether you are an individual contributor or represent an organization, we invite you to join us in shaping the future of this project. Together, we can build something truly remarkable.
```

Comment:
I don't think this needs to be deleted. We are still open-sourcing right?

Reply:
Microsoft open-source statement: https://github.com/microsoft/autogen/tree/main?tab=readme-ov-file#contributing

Reply:
@yiranwu0 -- this text is redundant with contributing/oss policy text at the bottom of the readme, which hasn't changed!